install-unix.sh

```diff
@@ -42,11 +42,11 @@ fi
 echo installling event_name=$1
 
 vm_cmd="cd ~/uber/ && ./run-simple-deploy.sh $1 && rm -f ./run-simple-deploy.sh"
-git clone https://github.com/magfest/ubersystem-deploy
+test -d ubersystem-deploy || git clone https://github.com/magfest/ubersystem-deploy
 cd ubersystem-deploy
```

Comment:
only issue here:  I'd prefer the behavior to be that if ubersystem-deploy already exists, we halt the install.  the reason is if we run run-simple-deploy.sh on an existing install, I'm not... 100% sure what's going to happen.  at the very least for now I'd call the undefined behavior.

the current behavior is that if ubersystem-deploy exists, git will error out with 'can't clone because ubersystem-deploy directory exists' which causes the entire thing to stop.

So, I'd say if we want to test if that dir exists, that's fine, but instead of continuing, we should still error out and say something like 
```
ERROR: Detected an install has already been completed/attempted, to re-install please: 
1) kill any vagrant instances in ubersystem-deploy with 'cd ubersystem-deploy && vagrant destroy'
2) remove ubersystem-deploy directory
3) run this install script again
```